### DISCUSSION TITLE
log4j vulnerability and Tandoor

### DISCUSSION THREAD

**Discussion by quamok** [TandoorRecipes/recipes#1127] [Q&A] [answered]
Hello, I'd like to know if the new CVE 10 log4j vulnerability affects Tandoor Recipes in any way. My guess is it does not, but I prefer to be safe and ask :)

source: https://nvd.nist.gov/vuln/detail/CVE-2021-44228

Thanks!

**Comment by smilerz** [comment, marked as answer]
Tandoor isn't using java anywhere so its good.
